source/patterns/@aws-solutions-constructs/resources/template-writer/lib/index.ts

```diff
@@ -16,7 +16,7 @@ import * as crypto from 'crypto';
 
 const s3Client = new S3Client({ region: process.env.REGION });
 
-export const handler = async (event, context) => {
+export const handler = async (event: any, context: any) => {
```

Comment:
Would you be up for running a `cdk-integ --no-cleanup` on one of the `aws-openapigateway-lambda` integ tests, just to verify that all still works as expected after changing the custom resource handler to be defined in Typescript? I want to make sure things like the index file don't end up in a different build/dist sub folder that didn't exist before. I'm happy to help verify behavior of the construct if you deploy it, just let me know!

Reply:
Refreshed the suite of integration tests for aws-openapigateway-lambda. Also confirmed proper functionality of `integ.opilam-apiFromAssetNewLambdaFunctions.js`.